src/asm/rpn.c

```diff
@@ -201,6 +201,34 @@ void rpn_BankSection(struct Expression *expr, char const *tzSectionName)
 	}
 }
 
+void rpn_SizeOfSection(struct Expression *expr, char const *tzSectionName)
```

Comment:
Please don't use Hungarian notation.

Reply:
It was to be consistent with all the other functions in rpn.c. Maybe a separate PR to do mass variable renaming?

Reply:
Sounds good.